example.py

```diff
@@ -20,10 +20,15 @@
 
 def print_scores(scores):
     for interviewer in scores.keys():
-        print("%s" % interviewer)
```

Comment:
Why move this inside the loop?

Reply:
My mistake, I just had it like that to keep the print statements together, but you're right, it should be called before the second loop. Fixed.